app/src/main/java/com/douglasbruce/tasky/features/register/RegisterScreen.kt

```diff
@@ -133,6 +137,10 @@ internal fun RegisterScreen(
                         isValid = uiState.isEmailValid,
                         placeholder = stringResource(R.string.email_placeholder),
                         supportingText = when (uiState.emailErrorType) {
+                            ErrorType.EMPTY -> {
+                                { Text(text = stringResource(R.string.email_error_empty)) }
```

Comment:
Nitpick, but I'm a friend of naming my string resources with `error` at the front, such as `error_email_empty`. It has the advantage that you get better code completion when you're looking for a specific error resource, but don't remember the exact name.

Reply:
Good point. Will make the change